Marlin/Marlin_main.cpp

```diff
@@ -176,7 +176,7 @@
  * M304 - Set bed PID parameters P I and D. (Requires PIDTEMPBED)
  * M350 - Set microstepping mode. (Requires digital microstepping pins.)
  * M351 - Toggle MS1 MS2 pins directly. (Requires digital microstepping pins.)
- * M355 - Turn the Case Light on/off and set its brightness. (Requires CASE_LIGHT_PIN)
+ * M355 - Sets Case Light brightness. (Requires CASE_LIGHT_PIN)
```

Comment:
For consistency, please phrase descriptions of G-codes as if they are prefaced with "This item will…" and not "This item…".

**BAD:** "Sets the frequency to 100 and opens the valve"
**GOOD:** "Set the frequency to 100 and open the valve"

I'm adding this guideline and others to the code and documentation style guides under construction.